test/functional/cmdLineTests/criu/playlist.xml

```diff
@@ -542,4 +542,27 @@
 			<impl>openj9</impl>
 		</impls>
 	</test>
+	<test>
+		<testCaseName>cmdLineTester_criu_softmx</testCaseName>
```

Comment:
Since this test is required to run in docker env, it should be disabled in the regular build. Please use `<disables>` to disable the test. For doc, please see https://github.com/eclipse-openj9/openj9/blob/master/test/docs/OpenJ9TestUserGuide.md#exclude-a-specific-test-variation

Reply:
Yes, we can modify the portable test's `testtarget` to integrate it. Since this test has many subtests, not sure if merging it together with existing portable test will make the docker checkpoint image larger with more snapshots and generated files?

Reply:
The ideal place is https://github.com/adoptium/aqa-tests/blob/master/external/functional-test/playlist.xml#L37, but we need to fix https://github.com/adoptium/aqa-tests/issues/4410 first.

Reply:
The ideal place is https://github.com/adoptium/aqa-tests/blob/master/external/functional-test/playlist.xml#L37, but we need to fix https://github.com/adoptium/aqa-tests/issues/4410 first

Reply:
https://github.com/adoptium/aqa-tests/issues/4410 has been fixed, when this test is ready, it can be added into the `TESTLIST` here https://github.com/adoptium/aqa-tests/blob/4285245463feb9edec768f2684d2e0e0462c2c09/external/criu-functional/playlist.xml#L18